### PR TITLE
[viewer] fix(BI): onViewerAction

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,11 @@
 ### :home: Internal
 - `inner-rce`
   - [#1579](https://github.com/wix-incubator/rich-content/pull/1579) create new contentState without version
+
+### :bug: Bug Fix
+- `viewer`
+  - [#1582](https://github.com/wix-incubator/rich-content/pull/1582) onViewerAction fix
```
Comment (mharsat) [inline CHANGELOG.md:25]:
```suggestion
  - [#1582](https://github.com/wix-incubator/rich-content/pull/1582) the helpers prop was not used, breaking BI callbacks
```

**Review by mharsat — APPROVED** [review]
This is a good reminder for the need to eliminate the usage of the soon deprecated helpers prop for passing BI callbacks

File: CHANGELOG.md (hunk 1 above)
Comment (Barackos) [reply CHANGELOG.md:25]: But it was used by 2 plugins.. Therefore - the break.
Not sure I understand what you mean.

Comment (mharsat) [reply CHANGELOG.md:21]: feel free to rephrase an explanatory changelog for consumers

**Pull request merged**